public/components/data_connections/components/data_connection.tsx

```diff
@@ -98,11 +91,19 @@ export const DataConnection = (props: any) => {
       id: 'connection_configuration',
       name: 'Connection configuration',
       disabled: false,
-      content: <></>,
+      content: (
+        <ConnectionDetails
+          allowedRoles={datasourceDetails.allowedRoles}
+          dataConnection={dataSource}
+          connector={datasourceDetails.connector}
+          properties={datasourceDetails.properties}
+        />
+      ),
     },
   ];
+  console.log(datasourceDetails);
```

Comment:
You can remove console.log

Reply:
Done in: [9ef64f2](https://github.com/opensearch-project/dashboards-observability/pull/1024/commits/9ef64f2dec4c065ad86757013dcaf092cb3cd72f)